lib/routes.js

```diff
@@ -1,3 +1,9 @@
+
```

Comment:
@bogdan Curiously, if I remove that line and run the specs, the CoffeeScript compiler continues to output that file with the blank line at the top. If you'd like, I suppose we could pass the output through `String#strip` here: https://github.com/railsware/js-routes/blob/master/spec/spec_helper.rb#L138